Assets/Scripts/UIAndMenus/MainMenu.cs

```diff
@@ -31,12 +32,5 @@ public void OnSettingsDoneButton()
     public void OnQuitButton()
     {
         Application.Quit();
```

Comment:
Quitting the application should eventually be done in a different class to handle the logic for different platforms (ex. it looks like we have to do something different if we have a web version) (also, while I can't currently think of a reason why you would quit from anywhere other than the main menu, it is possible that we may)

Maybe a QuitHandler class belonging to the GameStateManager that can be called through the GameStateManager?

Reply:
Agreed, I'm gonna make a note of this in the save system work item and push it to there